package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "bleno",
-  "version": "0.5.0",
+  "name": "@abandonware/bleno",
+  "version": "0.5.1-0",
```

Comment:
sorry I prefer to stay in 0.5.0 line in this case 0.5.0-1 would be better see other released modules in npm

Reply:
Ok. I'll do that!